File: strings/nameDecorator.py
# Robert Nelson, Name Decorator - Python

name = input("What is your first name?\n").strip().lower().capitalize()

smiley ="(: (: (: "
moreSmiley = " :) :) :)"
newString = smiley + name + moreSmiley
print(f"Hello {newString}, welcome to my program!")

File: strings/nameDecorator.c
// Robert Nelson, Name Decorator - C

#include <stdio.h>
#include <string.h>
char name[20];
int main(void){

    printf("Welcome, what is your name: \n");
    scanf("%s", name);
    char smiley[] = "(: (: (: ";
    char moreSmiley[] = " :) :) :)";
    //printf("%s", smiley);
    strcat(smiley, name);
    // printf("%s", smiley);
    strcat(smiley, moreSmiley); // can only concatanate one thing at a time
    // printf("%s", smiley);


    printf("Hello %s, welcome to my program!", smiley);

    return 0;
}

